### PR TITLE
163/login/projects model

### LINKED ISSUE #163: creating projects
[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)

**User Story**

[//]: # (Be descriptive about your user story. Who requests a feature and what's his intent?)

As an employee I want to be able to create new projects in our tool to represent projects in real life. For this I want a separate page where I can set a name for the project and text field that I can put the purpose of the project. Also I want to be able to pick a responsible person (there needs to be at least one). The default-responsible person should be the one creating the new project. There should be a submit-button in order to create the project.

When the submit button is clicked, I want to be redirected to the all-projects page.

**Acceptance Criteria:**

- [x] A page exists that allows creation of projects
- [x] A field for name of the project, description and responsible person exist
- [x] If name and description are not given, there are error messages and submitting does not work
- [x] At least one responsible person must be given for the button "create" to be clickable
- [x] The default responsible person is the person creating the project
- [x] clicking submit leads to the all-projects page #164 

[//]: # (Please don't forget to add the priority and complexity label to this issue.)

### PR DESCRIPTION
**Pull request by lassesteffen**
Fixes #163

### CODE REVIEW

File: spec/features/projects/new.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          expect(page).to have_text "Description can't be blank"
+        end
+      end
+    end
+
+    context 'when there is no responsible user selected' do
+      before do
+        fill_in 'project[name]', with: 'Name'
+        fill_in 'project[description]', with: 'Description'
+        page.unselect employee.email, from: 'project[responsible_user_ids][]'
```
Comment (janwestphal306) [inline spec/features/projects/new.html.erb_spec.rb]: Shouldn't this be `employee.id` instead of `employee.email`?

Comment (lassesteffen) [reply spec/features/projects/new.html.erb_spec.rb]: No, the employee.email is used as the visible value for the select and needs to be used for capybara

File: spec/features/projects/new.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    end
+
+    context 'when every field is correctly filled' do
+      before do
+        fill_in 'project[name]', with: 'Name'
+        fill_in 'project[description]', with: 'Description'
+        page.select employee.email, from: 'project[responsible_user_ids][]'
+      end
+
+      context 'when clicking the submit button' do
```
[the remark this answers is not in the document]
Comment (lassesteffen) [reply spec/features/projects/new.html.erb_spec.rb]: I agree

**Comment by lassesteffen** [comment]
@janwestphal306 fixed all the suggestions. I am only allowing empoyees right now, because there is an issue which allows admins to access all employee actions.

**Pull request merged**